index.js

```diff
@@ -3,6 +3,8 @@ const express = require('express');
 const s3o = require('s3o-middleware');
 const helmet = require('helmet');
 const express_enforces_ssl = require('express-enforces-ssl');
+const path = require('path');
+const { copy } = require('copy-paste');
```

Comment:
Why the `{  }`? I think `const copy = require('copy-paste').copy` gives a little more insight into how the module works/is being used by the app.